Lib/test/test_asyncio/test_streams.py

```diff
@@ -796,10 +794,7 @@ def test_drain_raises(self):
 
         def server():
             # Runs in a separate thread.
-            sock = socket.socket()
-            with sock:
-                sock.bind(('localhost', 0))
-                sock.listen(1)
+            with socket.bind_socket(('localhost', 0)) as sock:
```

Comment:
I would prefer to keep an explicit backlog=1... I'm not sure why listen(1) was used, so in case of doubt, I prefer to not change it :-)

Reply:
I'm pretty sure it makes no difference. Basically all other tests do not use the backlog.